ai-cfia-ia-acia/docs/about/team/cs.md

```diff
@@ -2,7 +2,7 @@
 sidebar_position: 2
 ---
 
-# Computer Science Team
+# Software engineering team
```

Comment:
https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case

In title case, capitalize the following words in a title or heading:

the first word of the title or heading, even if it is a minor word such as “The” or “A”the first word of a subtitlethe first word after a colon, em dash, or end punctuation in a headingmajor words, including the second part of hyphenated major words (e.g., “Self-Report,” not “Self-report”)words of four letters or more (e.g., “With,” “Between,” “From”)

Reply:
Thanks for the note on APA style for title case. I'll make sure to capitalize "The." However, I'd like to raise a concern about using "Engineering" in our team title. Based on [Article 22 of the Code des professions du Québec](https://www.legisquebec.gouv.qc.ca/fr/document/lc/I-9?langCont=en#se:22), only registered members of the Ordre des ingénieurs du Québec (OIQ) can legally use the title "Engineer." or engage in engineering activities. To stay legally compliant and avoid misrepresenting our qualifications, I propose we opt for "Development" instead of "Engineering" in our title.

What do you think?

Reply:
Yes, you are 100% correct @ThomasCardin

Reply:
...but then, how do we abbreviate? swe is software engineering which is used in ``swe.md```.

Reply:
sdt.md ?

Reply:
No, we don't wait for ADR to merge to make changes elsewhere. ADR tend to take weeks to developer and feed on our actual ongoing work. Waiting for ADR would turn our development process into a waterfall which we definitely don't want. Focus on providing incremental value to your users (in this case, developers).